FirstDemoAPI/BookService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using FirstDemoAPI.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace FirstDemoAPI
+{
+    public interface IBookService
+    {
+        List<Book> Get();
+        BookResponse Get(int id);
+        BookResponse Post(Book book);
+        BookResponse Put(int id, Book book);
+        void Delete(int id);
```

Comment:
Delete should return something that can confirm if delete operation is successful or failed